apps/portals/src/_Markdown.scss

```diff
@@ -34,7 +34,6 @@
     padding: 0;
   }
 
-  a,
```

Comment:
This style was setting the text color of `a` buttons to the primary color when it is now handled by the MUI Button component. I suspect, but have not confirmed, that this style was not applied to buttons before because of more specific CSS that has since been removed.